build/linux/installer/conf/td-agent-bit.conf

```diff
@@ -68,6 +68,19 @@
     Skip_Long_Lines On
     Ignore_Older 2m
 
+[INPUT]
+    Name tail
+    Tag oms.container.log.flbplugin.syslogstatus.*
+    Path /var/opt/microsoft/docker-cimprov/state/syslog.status
```

Comment:
why do we need this since you have already added  telemetry to track syslog enabled or not?

Reply:
will be removed